portfolio/src/main/webapp/script.js

```diff
@@ -24,3 +24,27 @@ function closeNav() {
 }
 
 
+function getMessage() {
+  fetch('/data').then(response => response.text()).then((message) => {
+    document.getElementById('message-container').innerText = message;
+  });
+}
+
+
+function getMessageJSON() {
+  fetch('/data').then(response => response.json()).then((message) => {
+    const messageListElement.innerHTML = '';
```

Comment:
Hmmm...I'm not sure this works. `messageListElement` should be a DOM object that you can modify to change how the page looks, but this statement is creating a new object.

Also, it doesn't look like you're invoking this function, which also suggests that this hasn't been tested. Even though we aren't using automated testing here, it's important to make sure that the code that gets merged into `master` works.

Reply:
In this case, a response to this comment would be really helpful to me, since you removed the DOM code in favor of logs. Something like: `Switched DOM code to logs since DOM modification is part of a later step.` helps me understand your intent.